tests/test_fixes.py

```diff
@@ -0,0 +1,69 @@
+import difflib
+from reccmp.isledecomp.compare.asm.fixes import find_effective_match
+
+
```

Comment:
Could you also add a test case asserting that `jg 0x1` vs. `jle 0x1` will not be accepted as effective?